src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -86,7 +86,7 @@ public static RestClientMethod BuildRequestMethod(InputOperation operation, Para
                 BuilderHelpers.EscapeXmlDescription(operation.Description),
                 responseType,
                 request,
-                parameters,
+                (operation.GroupParameters ? parameters.Select(p => (p != KnownParameters.WaitForCompletion && p != KnownParameters.RequestContext) ? p with { IsPropertyBag = true } : p ).ToArray() : parameters),
```

Comment:
We could reomve the outside parathesis
BTW could we add some comments explaining what this line is doing?

Reply:
added some comments